service/src/main/java/bio/terra/pipelines/db/entities/Pipeline.java

```diff
@@ -81,8 +81,8 @@ public Pipeline(
       String description,
       String pipelineType,
       String wdlUrl,
```

Comment:
do we think it doesn't make sense to rename wdlUrl as well?

Reply:
i think that field is actually specific to a wdl pipeline so i think its ok to keep that as is.  Or we rename it if we feel like a url is necessary to set up the config for different types of pipelines but i cant think of what url like CAS would need for setup.  I think eventually we'll want a whole table for each type of pipeline and stuff liek wdlUrl would go there but no need to split that out now until we see a second pipeline type

Reply:
makes sense to me, thanks!